transactions/remove-approved-nodes/2024/oct-1/README.md

```diff
@@ -0,0 +1,17 @@
+# Remove nodes from the Approved Node Operators ID List
+
+> Oct 1st, 2024
+
+See [FLIP GOV-291](https://github.com/onflow/flips/pull/292)
```

Comment:
Will be submitting this transaction only if the FLIP is approved by then. I have messaged Blockdaemon and Metrika